notifications/pkg/command/server.go

```diff
@@ -31,11 +31,17 @@ func Server(cfg *config.Config) *cli.Command {
 			}
 
 			evtsCfg := cfg.Notifications.Events
-			client, err := server.NewNatsStream(nats.Address(evtsCfg.Endpoint), nats.ClusterID(evtsCfg.Cluster))
+			client, err := server.NewNatsStream(
+				natsjs.Address(evtsCfg.Endpoint),
+				natsjs.ClusterID(evtsCfg.Cluster),
+			)
 			if err != nil {
 				return err
 			}
-			evts, err := events.Consume(client, evtsCfg.ConsumerGroup, evs...)
+			evts, err := events.Consume(
+				client, evtsCfg.ConsumerGroup,
+				evs...,
```

Comment:
Curious: Do we have an agreement when to use multi line function calls and when to do it in one line?